test/pom.xml

```diff
@@ -56,12 +56,6 @@ THE SOFTWARE.
         <artifactId>ant</artifactId>
         <version>1.11</version>
       </dependency>
-      <dependency>
```

Comment:
Maybe it makes sense to keep it so that we do not run into the issues in the future

Reply:
Maybe, but are we sure that the current `<dependencyManagement>` section will always apply to future `requireUpperBoundDeps` errors? It may or may not, so I think it is best to re-evaluate on a case by case basis. I think the ideal permanent solution would be to start using the plugin BOM, and/or remove plugin dependencies from the Jenkins core test suite.